### ISSUE TITLE
Design - Use all the widow for a large screen

### ISSUE THREAD

**Issue by decembre** [userstyles-world/userstyles.world#43]
I have one and before i make an userstyle for that, 
it's possible to use all the empty place (when we use a large monitor) ?
I hate to scroll  if it not necessary...

In the same idea, infos displayed can be reorganized when we use a large monitor:
As i said before in an other topic, by example for the userstyle info page:

TOP: 
Tittle and user name

LEFT:
Screenshot
Detail / Description under the screenshot

RIGHT:
Notes: with a limited height (height of the left side + overflow-y: auto)

POPUP (open on a click):
Code source

**Comment by vednoc** [comment]
I would prefer leaving source code as-is, so that users are more encouraged to inspect the source before clicking on install button. As for other suggestions, you could make a demo userstyle and we'll have a look at it! Aside from contributing directly, that's probably the easiest and the best way to merge changes suggested by the community.

Also, few days ago I ran a poll on our Discord server and we had 8 users for using more of the available space and 1 against it. We need to determine which maximum width would work best (and add others to USw Tweaks) because of this:

![3440x1440 screenshot](https://media.discordapp.net/attachments/827600537171656705/829039400121532486/unknown.png)

**Comment by decembre** [comment]
The demo  screenshot seems good, where is the problem ?

**Comment by Gusted** [comment]
>  where is the problem ?

It's messy now and I don't find it really easy to find for a particular style.

**Comment by decembre** [comment]
Yeah, but if we can have a sorting function for the explore page, it should be better.

For the usertyle page, i make a quick test for my wide screen:
![USw - TEST01 - 2021-04-10_160414](https://user-images.githubusercontent.com/868778/114272486-aa2d8500-9a16-11eb-8380-12411082ba2b.jpg)

The code is visible on hover " Source Code":
Many users don't take care of the code.....

Note:
it's a little bit strange this use of  selectors  which need to escaped as:
`.preview + .mt\:l`

Why not use  class without these ":"

**Comment by vednoc** [comment]
> i make a quick test for my wide screen

It looks perhaps a bit too compact. There are more buttons near "details" for moderators/administrators, as well as "Project homepage" button that isn't set in your userstyle. As for having 10 style cards on a single row (as in my screenshot), it's overwhelming, so we will have to add max-width (i.e. 1280px).

> Why not use class without these ":"

It's one of the things I'm working on. Utility classes were used for quick prototyping.

**Comment by decembre** [comment]
<blockquote>It looks perhaps a bit too compact:</blockquote>
I think it's better for a quick look and less scrolling ...

I style the "br" to reduce space.
And  (for a large window) i organize the page to read at once all the principal infos and not scrolling.

If i scroll, (by example to read an huge Note, 
i have always the screenshot to compare infos provided by it.

Many users don't take time to read too many infos:

Its my experience, by example, even if i notice  that an userstyle is obsolete, they install it...
;-)

<blockquote>Utility classes were used for quick prototyping.</blockquote>
Ok... was funny to plat with...
As it's a working progress:
Can you add a specific div for all the block with Author/ licence etc... (actually that's just individual "p""):
Like that, i think  i can move it under the screenshot, to let more space for Description / Details / Install

**Comment by decembre** [comment]
Here another possibility for the Explore / Search:
- Sticky header
- Large card with preview on left and Description on the right...

Should be more readable and less confuse... i hope.
![USw - Expore - Large card - TEST 1GIF 10-04-2021 23-20-24](https://user-images.githubusercontent.com/868778/114284983-05ca3380-9a54-11eb-803d-1399fb8c3961.gif)

**Comment by vednoc** [comment]
> I think it's better for a quick look and less scrolling

It varies from user to user. I prefer scrolling to find the information (most often I use find functionality if the page is long), and having to scroll in small boxes for Description/Notes would annoy me so much.

Hovering over crucial bits of information doesn't work either. It's very annoying, unless it's used for something like sidebars with links/icons, content that never changes (for the most part).

> Can you add a specific div for all the block with Author/ licence etc...

I'll be refactoring all of UI after the next update goes live, so all elements will have proper class names added to them. Some areas won't, like generated Markdown for Description/Notes/Biography.

> Should be more readable and less confuse... i hope.

Sticky search bar would be useful. As for cards, the images are too small and there is a lot of wasted space for style names/authors. I wouldn't go below 300px width when it comes to thumbnails.

**Comment by decembre** [comment]
<blockquote>It varies from user to user. </blockquote>
My taste is if i can have the max infos present at once in one screen, i can explore it quickly and if something keep my eye, i can focus on it.

Actually many site have a design Mobile centred and they don't take care to Desktop / Large Screen.

So all these waste spaces and vertical presentation (and scroll...).

But anyway, ok, that's you the master!

If need, i can make my Userstyle for my taste!
:-)

<blockquote>Hovering over crucial bits of information  doesn't work either</blockquote>
**That's true, but here that's about Large screen.**

I try to adjust the size of these elements to show the max of infos to not have to scroll.
By example:
- Description / Details don't need many place;
Short height
- Notes can be huge (as mine sometime):
Need more space but limited.
If i am interested by the beginning, i scroll inside.
Etc...

<blockquote> As for cards, the images are too small</blockquote>

Yep, but if i use this CSS , i can enlarge them on hover with an addon (as Imagus or Thumbnail Zoom Plus):
<code>
.card .thumbnail figure::after {
   pointer-events: none !important;
}

.card.col figure::before {
    pointer-events: none !important;
}</code>

**Comment by vednoc** [comment]
> So all these waste spaces and vertical presentation (and scroll...).

Not necessarily. Content is usually ordered in order of importance, which is also the case with USw.

There is a reason why dashboards try to fit in as much information as possible, meanwhile blogs and text-heavy websites _usually_ have a lot of whitespace: the former is a lot more useful with everything on the screen, and the latter is a lot more readable and less confusing when there is a clear separation between sections.

Don't get me wrong, I don't like how it is currently. This is me challenging design decisions, so that we can find something that would work for most users, not just some of the users.

> If need, i can make my Userstyle for my taste!

And that's what I'd love to see. Especially after I add proper selectors to the UI.

> I try to adjust the size of these elements to show the max of infos to not have to scroll.

While I agree and appreciate that, it could be hard to predict all use-cases. Sometimes it's near impossible to do that.

> i can enlarge them on hover with an addon

Welcome to 1% club that uses one of the mentioned extensions. 😄 That's exactly why small thumbnails would be a great fit for custom userstyles.

**Comment by decembre** [comment]
I am waiting the next update to play more with your site!
I see that your are selected by Stylus addon to figure in their inline search:
That's a good new....

**Comment by decembre** [comment]
Another idea:
On Account pages, put the "Welcome" and "Info / Edit settings etc" in top header in Tabs (visible on hover or click ).

Here my test:
![USw - Widesceen - TESt - TAB options - TOP-LEFT - GIF 29-05-2021 10-33-04](https://user-images.githubusercontent.com/868778/120066942-42d89c80-c079-11eb-9b41-ba98727eb45b.gif)
.
Like that we have all the place for the Userstyles cards.
